### DISCUSSION TITLE
App published date?

### DISCUSSION THREAD

**Discussion by zawiya** [Budibase/budibase#9005] [Help]
Hi I was wondering if there was a way I could access the last Published date as a binding within the app?
I wanted to show a "Last Updated" component.

**Comment by andz-bb** [comment]
Hey @zawiya - I had a look in the code and unfortunately I don't see anywhere where the deployment information in the portal could be accessed by the app itself.

A workaround I thought of was to add a REST API datasource and connect it to the Budibase public API. When you hit the [get application endpoint](https://docs.budibase.com/reference/get_applications-appid) it returns the date/time when the application was last updated in the builder. You can then use the result from that endpoint within the application - hope this helps!
